torch/__init__.py

```diff
@@ -1442,10 +1442,13 @@ def fn(model: Callable):
     import torch._dynamo
     if mode is not None and options is not None:
         raise RuntimeError("Either mode or options can be specified, but both can't be specified at the same time.")
+    if torch.backends.mps.is_available():
+        backend = "aot_eager"
     if mode is None and options is None:
-        mode = "default"
+        mode = "default" 
```

Comment:
lint might not be happy about that :D

Reply:
Just checked after `pip install lintrunner -a` and no lint issues